backend/app/core/venue_service.py

```
@@ -229,311 +264,398 @@ def create_facility(self, facility_data: dict) -> Facility:
-    def get_venue_reviews(self, venue_id: int, limit: int = 20, offset: int = 0) -> Tuple[List[VenueReview], int]:
+
+    def get_venue_reviews(
+        self, venue_id: int, limit: int = 20, offset: int = 0
+    ) -> Tuple[List[VenueReview], int]:
         """Get reviews for venue"""
         query = self.db.query(VenueReview).filter(
-            and_(
-                VenueReview.venue_id == venue_id,
-                VenueReview.is_public == True
-            )
+            and_(VenueReview.venue_id == venue_id, VenueReview.is_public == True)
```

Ruff (0.11.9)

Avoid equality comparisons to `True`; use `if VenueReview.is_public:` for truth checks

Replace with `VenueReview.is_public`

(E712)